tests/nix/nat-testing.nix

```diff
@@ -0,0 +1,128 @@
+# here is the testing base file: https://github.com/NixOS/nixpkgs/blob/master/nixos/lib/testing-python.nix
+with import <nixpkgs> {};
```

Comment:
Should end up using `./pkgs.nix`. Will need to setup something proper with Tetra and Matrix Agent to get this working. Then paths like `<nixpkgs/...` shouldn't be used since that's not using the same path. Got to use `pkgs.path` instead. Substrate involves alot of this.

Reply:
@nzhang-zh

Reply:
yeah so the `pkgs.nix` has been changed to a function and I have just changed this nix expression to be using that.